src/inmanta/env.py

```diff
@@ -101,6 +101,21 @@ def _init_env(self) -> bool:
                 return False
             LOGGER.debug("Created a new virtualenv at %s", self.env_path)
 
+        else:
```

Comment:
These changes do not correspond with the changes in https://github.com/inmanta/inmanta-core/commit/4e889fe930c811dc5951d3cce5d020dcb45ce6f4. Why?

Reply:
There is no `_path_pth_file` logic in iso4, should I have added it ? I don't truly understand what is going on here, I think I need some help to finish this ticket.